fs/blktests.py

```diff
@@ -33,8 +33,10 @@ def setUp(self):
         '''
         Setup Blktests
         '''
+        self.disk = []
         self.disk = self.params.get('disk', default='')
         self.dev_type = self.params.get('type', default='')
+        self.disk = self.disk.split(',')
```

Comment:
@vrbagalkot can we make this as space aligned? to be common with other test cases.